function_app.py

```
@@ -0,0 +1,221 @@
+    """
+    try:
+        # Get the uploaded file
+        file = req.files.get('image')
+        
+        if not file:
+            # Try to get from body as base64
+            try:
+                body = req.get_json()
+                image_data = body.get('image')
+                filename = body.get('filename', 'uploaded_image.jpg')
+            except:
```

Ruff (0.14.7)

Do not use bare `except`

(E722)